assignments/this.js

```diff
@@ -1,26 +1,53 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
 *
-* 1. 
-* 2. 
-* 3. 
-* 4. 
+* 1. When you use this in a global aspect, it will refer to the window or console.
```

Comment:
Another way you could say this is that if it's not bound to a specific object, it will refer to the _global object_, which is the `window` in browsers and the `console` in Node.js.